comment/validators.py

```diff
@@ -121,3 +122,16 @@ def validate(self, request):
 
 class ValidatorMixin(ContentTypeValidator, ParentIdValidator):
     pass
+
+
+class DABEmailValidator(EmailValidator):
+    def __init__(self, email):
+        super().__init__(EmailError.EMAIL_INVALID)
+        self.email = email
+
+    def is_valid(self):
+        try:
+            self(self.email)
+            return True
+        except ValidationError:
+            return False
```

Comment:
why are we not using the default [`EmailValidator`](https://docs.djangoproject.com/en/dev/ref/validators/#emailvalidator) when we aren't performing any custom validation? The error message can be overriden if required through `EmailValidator(message=my_message)`. 

Although in our case, we don't necessarily need to do this as the default error message is sufficient `Enter a valid email address`.

Reply:
Hmmm, you have previously asked me to move the validitaor to a custom one 😝.
Using the custom validator is cleaner than writing try block inside the view and our vlidation message is identical to the default one, I added it to the message class so that it can be added to the custom translations.

Reply:
> I added it to the message class so that it can be added to the custom translations.

You can probably trust `django` to have more translation coverage than our app at the moment :smile:

Reply:
😒 I do. please read my statement again. I said that the message can be added to **custom translations.** So that the developer can choose the suitable translation and I believe django or any other app doesn't completely fit with the tastes of all people.